egs/commonvoice/ASR/pruned_transducer_stateless7/train.py

```diff
@@ -1027,7 +1050,13 @@ def run(rank, world_size, args):
 
     commonvoice = CommonVoiceAsrDataModule(args)
 
-    train_cuts = commonvoice.train_cuts()
+    if not args.use_validated_set:
+        train_cuts = commonvoice.train_cuts()
+    else:
+        train_cuts = commonvoice.validated_cuts()
```

Comment:
```suggestion
    if args.use_validated_set:
        train_cuts = commonvoice.validated_cuts()
    else:
        train_cuts = commonvoice.train_cuts()
```

It is easier to reason without `not`.